### PR TITLE
Fix MM 'offset.commit.interval.ms' option

### PR DESCRIPTION
**Pull request by stanlyDoge**
### Type of change
- Bugfix

### Description
https://github.com/strimzi/strimzi-kafka-operator/issues/1734

### Checklist

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java [hunk 1]
```diff
@@ -97,6 +99,16 @@ public void setWhitelist(String whitelist) {
         this.whitelist = whitelist;
     }
 
+    @Description("Offset commit interval in ms. Default value is 60 000.")
+    @DefaultValue("60000")
```
Comment (scholzj) [inline api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java]: Should we set also `@JsonInclude(JsonInclude.Include.NON_DEFAULT)`?

**Comment by stanlyDoge** [comment]
@scholzj we can add `num.streams` for sure. Any other option we want in this PR?

**Comment by scholzj** [comment]
@stanlyDoge You probably know more about Mirror Maker than me. So I leave it up to you which others we should support as well. But we should try to do it all at once, not wait until the next user opens an issue. From a brief look it seemed most of the other options might have required some additional classes which will be complicated with Strimzi. So maybe all we need to add is the `num-streams`. But I'm not 100% sure.

**Comment by ppatierno** [comment]
We already have support for `num.streams` in the consumer section.

File: docker-images/kafka/scripts/kafka_mirror_maker_run.sh [hunk 2]
```diff
@@ -71,11 +71,23 @@ if [ -n "$KAFKA_MIRRORMAKER_NUMSTREAMS" ]; then
     numstreams="--num.streams ${KAFKA_MIRRORMAKER_NUMSTREAMS}"
 fi
 
+if [ -n "$KAFKA_MIRRORMAKER_OFFSET_COMMIT_INTERVAL" ]; then
+    offset_commit_interval="--offset.commit.interval.ms $KAFKA_MIRRORMAKER_OFFSET_COMMIT_INTERVAL"
+fi
+
```
Comment (scholzj) [inline docker-images/kafka/scripts/kafka_mirror_maker_run.sh]: Would just one blank line be enough?

File: docker-images/kafka/scripts/kafka_mirror_maker_run.sh [hunk 3]
```diff
@@ -71,11 +71,23 @@ if [ -n "$KAFKA_MIRRORMAKER_NUMSTREAMS" ]; then
...
+
+if [ -n "$KAFKA_MIRRORMAKER_ABORT_ON_SEND_FAILURE" ]; then
+    abort_on_send_failure="--abort.on.send.failure $KAFKA_MIRRORMAKER_ABORT_ON_SEND_FAILURE"
+fi
+
```
Comment (scholzj) [inline docker-images/kafka/scripts/kafka_mirror_maker_run.sh]: Would just one blank line be enough?

File: docker-images/kafka/scripts/kafka_mirror_maker_run.sh (hunk 2 above)
Comment (stanlyDoge) [reply docker-images/kafka/scripts/kafka_mirror_maker_run.sh]: Yep

**Comment by stanlyDoge** [comment]
Regarding docs - should I or { @PaulRMellor | @laidan6000 } do this? 
I am for doing it in another PR.

**Comment by scholzj** [comment]
@stanlyDoge I guess it depends how complicated it would be ... adding two more options to the existing docs sounds normally quite simple for you to do it, but I haven't seen this particular docs in detail, so up to you I guess. Either way we need to make sure someone does it ;-).

File: api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumerSpec.java [hunk 4]
```diff
@@ -58,4 +60,14 @@ public String getGroupId() {
     public void setGroupId(String groupId) {
         this.groupId = groupId;
     }
+
+    @Description("Offset commit interval in ms. Default value is 60 000.")
```
Comment (ppatierno) [inline api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumerSpec.java]: why a space in "60 000"?

Comment (stanlyDoge) [reply api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumerSpec.java]: It is fixed in https://github.com/strimzi/strimzi-kafka-operator/pull/1768/files#diff-d79dc54e295cffd73c71facebff75566R64

Comment (ppatierno) [reply api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumerSpec.java]: I don't know why github showed me the old version! :-)

Comment (stanlyDoge) [reply api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumerSpec.java]: It is another PR. I had to update the `spec` file to apply suggested changes in the docs.

**Pull request merged**